server/src/main/java/io/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -835,11 +835,12 @@ private void updatePayload(final Handle handle, final DataSegment segment) throw
 
   public List<DataSegment> getUnusedSegmentsForInterval(final String dataSource, final Interval interval)
```

Comment:
MetadataSegmentManager isn't registered on the indexer. are you talking about as a static method, or making the sql stuff more abstracted so that the indexer can also use it?